db/seed.sql

```diff
@@ -3168,7 +3166,7 @@ CREATE TABLE `cron_schedule` (
   PRIMARY KEY (`schedule_id`),
   KEY `CRON_SCHEDULE_JOB_CODE` (`job_code`),
   KEY `CRON_SCHEDULE_SCHEDULED_AT_STATUS` (`scheduled_at`,`status`)
-) ENGINE=InnoDB AUTO_INCREMENT=509 DEFAULT CHARSET=utf8 COMMENT='Cron Schedule';
+) ENGINE=InnoDB AUTO_INCREMENT=1 DEFAULT CHARSET=utf8 COMMENT='Cron Schedule';
```

Comment:
Got sick of all this junk in the database seed, so it's removed (several tables).